### PR TITLE
update the translation of `Object.values()`

### PR DESCRIPTION
**Pull request by robert-jx**
### Description

update the translation of `Object.values()`
### Motivation

update the translation of `Object.values()`
### Additional details

none

### CODE REVIEW

File: files/zh-cn/web/javascript/reference/global_objects/object/values/index.md [hunk 1]
```diff
@@ -5,49 +5,73 @@ slug: Web/JavaScript/Reference/Global_Objects/Object/values
 
 {{JSRef}}
 
-**`Object.values()`** 方法返回一个给定对象自身的所有可枚举属性值的数组，值的顺序与使用 {{jsxref("Statements/for...in", "for...in")}} 循环的顺序相同（区别在于 for-in 循环枚举原型链中的属性）。
+**`Object.values()`** 静态方法返回一个给定对象自身的可枚举字符串键属性值组成的数组。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/global_objects/object/values/index.md:8]:
```suggestion
**`Object.values()`** 静态方法返回一个给定对象的自有可枚举字符串键属性值组成的数组。
```

File: files/zh-cn/web/javascript/reference/global_objects/object/values/index.md [hunk 2]
```diff
@@ -5,49 +5,73 @@ slug: Web/JavaScript/Reference/Global_Objects/Object/values
...
 ### 参数
 
 - `obj`
-  - : 被返回可枚举属性值的对象。
+  - : 一个对象。
 
 ### 返回值
 
-一个包含对象自身的所有可枚举属性值的数组。
+一个包含了给定对象自身的可枚举字符串键属性值的数组。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/global_objects/object/values/index.md:25]:
```suggestion
一个包含了给定对象的自有可枚举字符串键属性值的数组。
```

**Review by JasonLamv-t — APPROVED** [review]
Some suggestions

**Review by yin1999 — APPROVED** [review]
:+1:

**Pull request merged**